packages/console/src/notebook/Editor.tsx

```diff
@@ -94,7 +94,11 @@ class Editor extends Component<EditorProps, Record<string, never>> {
       ...settings,
     };
     assertNotNull(this.container);
+
+    MonacoUtils.removeConflictingKeybindings();
```

Comment:
We could just call this from `MonacoUtils.init`, and then don't need to check in the function if it's been called more than once.